hal-x86_64/src/interrupt/mod.rs

```diff
@@ -58,7 +58,7 @@ impl Handlers<crate::X64> for TestHandlersImpl {
     {
         let mut vga = vga::writer();
         vga.set_color(vga::ColorSpec::new(vga::Color::Red, vga::Color::Black));
-        writeln!(&mut vga, "page fault\n{:#?}", cx.registers()).unwrap();
+        log::error!("page fault\n{:#?}", cx.registers());
         vga.set_color(vga::ColorSpec::new(vga::Color::Green, vga::Color::Black));
```

Comment:
Given that the Logger now handles adding `[LEVEL]` tags & doing colouring, we can probably get rid of the call to `vga::writer()` and the calls to `vga.set_color` entirely.